README.md

```diff
@@ -8,154 +8,66 @@ The ArkCase architecture is described here: https://www.arkcase.com/developer-su
-* VirtualBox <https://www.virtualbox.org>
-* Vagrant <https://www.vagrantup.com>
-* Tomcat 9 <https://tomcat.apache.org>
-* git <https://git-scm.com/>
-* nodejs <https://nodejs.org>
-    * MacOS: install Node 6.  Node 8 and Node 11 do not work on MacOS.  
-    * Windows and Linux: use Node 8 or above.
-* npm (comes with NodeJS)
-* yarn <https://yarnpkg.com>
-
-### Build the Vagrant VM
-
-In this section you will build the Vagrant VM which will run the services ArkCase requires.  These services include Solr, ActiveMQ, MySQL, Alfresco, and Pentaho. 
-
-First, install all the prerequisites (see Prerequisites section above).
-
-Next, build the Vagrant VM according to the instructions in the `arkcase-ce` repository: <https://github.com/ArkCase/arkcase-ce>.
-
-After the box is up, the following URLs should work from your browser; be aware that ArkCase uses a self-signed TLS certificate, so you will have to accept the browser warning about the unrecognized, self-signed certificate. 
-
-https://arkcase-ce.local/solr
-
-https://arkcase-ce.local/share
-
-https://arkcase-ce.local/pentaho
-
-https://arkcase-ce.local/VirtualViewerJavaHTML5 (expect a 503 error from this URL)
-
-### Clone the repository and build the war file
-
-Clone this repository to a folder of your choice.
-
-`cd` to the root folder of this repository; then run `mvn -DskipITs clean install`.  This will run the unit tests and build the war file.  It should take a few minutes.
-
-### Clone the configuration folder
-
-ArkCase requires a configuration folder which is housed in another GitHub repository: https://github.com/ArkCase/.arkcase; follow the instructions at this link to setup the configuration folder.
-
-### Run the Configuration Server
-
-Starting with version 3.3.1, ArkCase requires a separate configuration server, based on Spring Cloud Config Server (more info here: https://spring.io/projects/spring-cloud-config).  To start the config server, take these steps:
-
-* Download the most recent config-server.jar file from here: https://github.com/ArkCase/acm-config-server/releases
-* Start the server process with this command: `java -jar config-server-0.0.1.jar`, replacing `0.0.1` with the version you downloaded.
-
-The config server runs on port 9999 by default.  To run on a different port, add the server.port option to the command, like so: `java -Dserver.port=8888 -jar config-server-0.0.1.jar`, replacing `8888` with the desired port.
-
-### Configure Tomcat
-
-#### Tomcat Native Connector
+- **Operating System**: [Ubuntu](https://ubuntu.com/download) (preferably 20.04 LTS)
+- [Helm 3.12.1](https://helm.sh/docs/intro/install/) for managing Kubernetes applications.(Versions above 3.13 are not yet supported)
+- A functioning [Kubernetes cluster](https://kubernetes.io/docs/setup/).
+- [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/) for interacting with your Kubernetes cluster.
+- [Docker](https://docs.docker.com/get-docker/) for containerization.
```

Comment:
Docker doesn't have to be mentioned either ... it will come along with Kubernetes.